src/components/cells/cell-menu-container.js

```diff
@@ -35,6 +38,23 @@ export class CellMenuContainerUnconnected extends React.Component {
 
   render() {
     const { anchorElement } = this.state
+    const includeInRunAllIndicator = (
```

Comment:
I think this is cool, and I might propose some pretty minor style changes to tighten it up, but I think this is the right direction. re: the cell menu, totally cool for it to not be that height, so long as the rest of it aligns well (border vs. outline, etc) which it does now. I agree that cells don't stay empty for long, but we do have a first impression to make.